### PR TITLE
a List

### PR DESCRIPTION
**Pull request by bazzika**

### CODE REVIEW

**Comment by nesterone** [comment]
Please, use autoformat feature of your IDE

File: 01-syntax/03-data-structures/a-list/arrayToList.js [hunk 1]
```diff
@@ -4,6 +4,13 @@
  */
 
 // Your code here.
+function arrayToList(arr){
+    var list=null;
```
Comment (nesterone) [inline 01-syntax/03-data-structures/a-list/arrayToList.js]: 'var list;' could be enough

File: 01-syntax/03-data-structures/a-list/listToArray.js [hunk 2]
```diff
@@ -4,6 +4,13 @@
  */
 
 //Your code here
+function listToArray(list) {
+    var arr = [];
+    for (var node = list; node; node = node.rest) {
+        arr.push(node.value);
+    }
+        return arr;
```
Comment (nesterone) [inline 01-syntax/03-data-structures/a-list/listToArray.js]: As I sad, use autoformatting to correct such stuff

File: 01-syntax/03-data-structures/a-list/nth.js [hunk 3]
```diff
@@ -5,6 +5,15 @@
 
 
 //Your code here
+function nth(list, number) {
+    if (!list) {
+    return undefined;}
```
Comment (nesterone) [inline 01-syntax/03-data-structures/a-list/nth.js]: 'return;' could be enoughts

File: 01-syntax/03-data-structures/deep-comparation/deepEqual.js [hunk 4]
```diff
@@ -6,10 +6,40 @@
 
 // Your code here.
 
+
+function deepEqual(obj1,obj2) {
+    if (obj1===obj2) {
+        return true;
+    }
+
+    /*if (typeof obj1 != 'object' || obj1==null || typeof obj2!='object' || obj2==null)
```
Comment (nesterone) [inline 01-syntax/03-data-structures/deep-comparation/deepEqual.js]: It's not a good practice to commit code with commented code

**Comment by nesterone** [comment]
Please, correct my concernes, most of them are about code style and formating

Take attention that your pull request is for task #57  don't mix it with deepEqual

File: 01-syntax/03-data-structures/a-list/arrayToList.js (hunk 1 above)
Comment (bazzika) [inline 01-syntax/03-data-structures/a-list/arrayToList.js]: I don't agree. If use your variant : { value: 10, rest: { value: 20, rest: undefined } };
if Var list = null :{ value: 10, rest: { value: 20, rest: null } }

**Pull request closed without merging**